src/System.Private.ServiceModel/tests/System.Private.ServiceModel.Tests.csproj

```diff
@@ -14,7 +14,7 @@
     <ProjectGuid>{BB3B114D-68D7-4E36-9EBE-127E694E6EC9}</ProjectGuid>
   </PropertyGroup>
   <ItemGroup>
-    <Compile Include="**\*.cs" Exclude="Scenarios\SelfHostWcfService\*" />
+    <Compile Include="**\*.cs" Exclude="Scenarios\SelfHostWcfService\**;Scenarios\Bridge\**" />
```

Comment:
Small merge conflict ahead: you may want to coordinate on this; https://github.com/dotnet/wcf/pull/120#discussion-diff-33398115R17 removes this in favour of excluding the entire Scenarios\ directory.